cmd/snap/cmd_run_test.go

```diff
@@ -465,7 +465,7 @@ func (s *SnapSuite) TestSnapRunIsReexeced(c *check.C) {
 }
 
 func (s *SnapSuite) TestSnapRunAppIntegrationFromCore(c *check.C) {
-	defer mockSnapConfine(filepath.Join(dirs.SnapMountDir, "core", "current", dirs.CoreLibExecDir))()
+	defer mockSnapConfine(filepath.Join(dirs.SnapMountDir, "core", "111", dirs.CoreLibExecDir))()
```

Comment:
I like how this is actually better than before because it keeps the revision "in sync" and is thus more reliable in case of transient symlink updates.